www/pfSense-pkg-vHosts/README.md

```diff
@@ -0,0 +1,58 @@
+# vHosts
```

Comment:
Please do not add other files like this in root port directory. This information can be moved as a comment in the source code